pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -773,8 +792,12 @@ func GatherCRD(i *Gatherer) func() ([]record.Record, []error) {
 			if err != nil {
 				return []record.Record{}, []error{err}
 			}
+			recordName := fmt.Sprintf("config/crd/%s", crd.Name)
```

Comment:
CRDs themselves are also Cluster scoped. The hint is also that Api doesn't offer an option to fetch it from namespace.